app/views/content/non-uk-teachers/non-uk-qualifications.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "Check your eligibility to train to teach in England"
```

Comment:
Could we make this simpler and say check your qualifications?

Reply:
@laurahermionetennant92 I cant find it in here to tag it but on the non-UK category it needs a blue H2 above the card. Maybe something along the lines of what qualifications you need? And the title on the card could be how to check your qualifications?

Reply:
It would need to be, 'Check the qualifications you'll need to train to teach in England'
because I think we saw quite a lot of confusion about whether we're referring to the quals you need to teach, or the quals you need to train, and where QTS fits into that. I think eligibility is well-understood – it hasn't come up in user research as a difficulty.
Re the quals card, I'm trying to move it below 'If you want to train to teach' but haven't worked out how yet!